mock/job.go

```diff
@@ -52,6 +52,14 @@ func (repo *ProjectJobSpecRepository) GetAll(ctx context.Context) ([]models.JobS
 	return []models.JobSpec{}, args.Error(1)
 }
 
+func (repo *ProjectJobSpecRepository) GetAllWithNamespace(ctx context.Context) (map[string][]string, error) {
```

Comment:
Why are we adding this?

Reply:
Removed this. Missed to delete this one previously.